includes/classes/class-plugin-manager.php

```diff
@@ -0,0 +1,409 @@
+<?php
+/**
+ * Class PerformanceLab\Plugin_Manager
+ *
+ * @package PerformanceLab
+ * Author: WordPress Performance Team
+ * Author URI: https://make.wordpress.org/performance/
+ * License: GPLv2 or later
+ * License URI: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+ */
+
+namespace PerformanceLab;
+
+use WP_Error;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * General purpose class for managing the UI of standalone performance plugins
+ * and rendering of the UI associated therewith.
+ *
+ * @since n.e.x.t
+ * @access public
+ * @ignore
+ */
+class Plugin_Manager {
+
+	/**
+	 * Get info about all the standalone plugins and their status.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array of plugins info.
+	 */
+	public static function get_standalone_plugins() {
+		$managed_standalone_plugins = array(
+			'webp-uploads',
+			'performant-translations',
+			'sqlite-database-integration',
```

Comment:
The SQLite plugin is no longer under the performance team umbrella, instead we need to add the recently released Dominant Color Images here.
```suggestion
			'dominant-color-images',
```